src/reference-manual/pathfinder.qmd

```diff
@@ -25,3 +25,29 @@ evaluations, with greater reductions for more challenging posteriors.
 While the evaluations in @zhang_pathfinder:2022 found that
 single-path and multi-path Pathfinder outperform ADVI for most of the models in the PosteriorDB evaluation set,
 we recognize the need for further experiments on a wider range of models.
+
+## Diagnosing Pathfinder
+
+Pathfinder diagnoses the accuracy of the approximation by computing the density ratio of the true posterior and 
+the approximation and using Pareto-$\hat{k}$ diagnostic (Vehtari et al., 2024) to assess whether these ratios can
+be used to improve the approximation via resmapling. /, the
```

Comment:
resmapling is misspelled.

I don't know what the / is doing here---should this be a different character?